packages/sdk/src/constants.ts

```diff
@@ -0,0 +1 @@
+export const SENTRY_DSN = "https://4659311eb5cb472982a5f7d392f3e2b4@sentry.io/1430767";
```

Comment:
I agree that it is definitely less than ideal having this there, but no easy solve right now. imo, the sdk package is not really necessary now that we are serving everything from an iframe and not directly on the site. I think the solve here is to just move all `react` stuff in sdk back in to the dapp. 

Make just throw a TODO in explaining some of the thought process?

Reply:
The SDK package is still used to build this (relatively) minimal JS script that does run directly on the newsroom site in order to run the iframe resizer script (also does some other stuff - the stuff that can generate errors Sentry needs to collect - that in retrospect should be refactored and moved into the iframe).

I'll add a TODO and also make a ticket for refactoring some stuff so that the logic that needs Sentry happens inside the iframe.